images/router/haproxy/reload-haproxy

```diff
@@ -1,21 +1,64 @@
 #!/bin/bash -xu
 
+script=$(basename "${BASH_SOURCE[0]}")
+DEFAULT_COALESCE_INTERVAL=15   #  default 15 secs.
+COALESCE_MAX_WAIT=$((5 * 60))  #  5 minutes
```

Comment:
Its up to you, I think the convention is lowercase for variable which are not from the outside or exposed to the outside (although in this case they are global, so you can't use local)   it's totally style....